tox.ini

```diff
@@ -6,10 +6,11 @@ setenv =
     PYTHONPATH = {toxinidir}:{toxinidir}/multinosetests
 commands =
     make check
-deps =
-    -rrequirements-dev.txt
+deps = -rrequirements.txt
```

Comment:
No, it doesn't, it isn't necessary. I was just surprised by the syntax without space.